### ISSUE TITLE
Other name?

### ISSUE THREAD

**Issue by koenvervloesem** [rhasspy/rhasspy-services#1]
Rhasspy-services doesn't seem to be the right name, as the README also refers to tools and libraries. Maybe just call it rhasspy if we're sure that we won't need that name for another repository?

**Comment by synesthesiam** [comment]
I keep going back and forth about this. Maybe I'll just call it `rhasspy` and then shift the services descriptions to the wiki.

**Comment by koenvervloesem** [comment]
I'm not sure why this should be a wiki?

**Comment by maxbachmann** [comment]
what is going to happen to the original rhasspy repository?

**Comment by synesthesiam** [comment]
I'm not sure yet what to do with the original repo once things move more over here. It's got a lot of stars, so I'd like to keep it. Any thoughts?

**Comment by koenvervloesem** [comment]
Wow, I didn't notice the number of stars, amazing!

I think keeping all repositories together in one place (this organization) is best for discoverability reasons. It seems all information will be transferred together with the original repository if you transfer it to an organization, at least that's how I interpret [the page about transfering a repository to an organization](https://help.github.com/en/github/administering-a-repository/transferring-a-repository):

> **What's transferred with a repository?**
> 
> When you transfer a repository, its issues, pull requests, wiki, **stars**, and watchers are also transferred. If the transferred repository contains webhooks, services, secrets, or deploy keys, they will remain associated after the transfer is complete. Git information about commits, including contributions, is preserved.

But there's one problem:

> The target account must not have a repository with the same name, or a fork in the same network.

**Comment by maxbachmann** [comment]
yes it transfers everything (already did that with some repos) and when moving a repo github will automatically redirect anyone still using the old url of the repo to the new one.  

Maybe "convert" it to a repo containing the stuff from rhasspy-voltron and the service stuff from here and keep the name rhasspy for it.

Oh and btw you can simply rename a repo aswell and keep all stuff like issues stars watchers ...

**Comment by 0x326** [comment]
@synesthesiam You could rename the [`synesthesiam/rhasspy`](https://github.com/synesthesiam/rhasspy) repo to `synesthesiam/rhasspy-tool`, then transfer the intermediate `synesthesiam/rhasspy-tool` to `rhasspy/rhasspy-tool`.

That would keep all of your stars on GitHub
